internal/node/controlapi_test.go

```diff
@@ -0,0 +1,81 @@
+package nexnode
+
+import (
+	"slices"
+	"testing"
+
+	controlapi "github.com/synadia-io/nex/control-api"
+)
+
+func TestSummarizeMachinesForPing(t *testing.T) {
```

Comment:
Do we not want to keep these tests packaged alongside the others?

Reply:
this is a unit test. It is exactly where a unit test belongs in Go, in a `_test.go` file sitting next to the file under test.

Reply:
Other unit tests are under the `test/` package right now. As of the current state of how things are being run, this test file is not being included in the suite when it runs.

Reply:
unit tests shouldn't be in `test/`. That's not the standard Go pattern. We run `go test -v ./...` in a root directory to fire off unit tests. Anything in the `test/` directory is acceptance/integration/etc and launched separately.

Reply:
I am not saying having unit tests live alongside their counterparts isn't correct I am just saying I am pretty certain we are not running this new test you just wrote.

Reply:
You need to modify how we are running the tests.

It is currently `go test -v -race ./test` in CI.

Reply:
that's how it runs the spec suite in `/test`. The unit tests are an entirely different concern and are usually run separately.

Reply:
This is how it runs the unit tests... not the spec suite...